articles/connections/social/soundcloud.md

```diff
@@ -2,7 +2,7 @@
 title: Connect Apps to SoundCloud
 connection: SoundCloud
 toc: true
-public: true
+public: false
```

Comment:
Removes this connection from the list but keeps the page live. This connection no longer has a record in the dashboard or in marketplace.